CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+# Contributing
+
+Pull requests are welcome!
+
+## Getting started
+
+First [fork this repo](https://help.github.com/articles/fork-a-repo/), and
+then clone it & set the upstream repo:
+
+```bash
+git clone git@github.com:your-github-username/halyard.git
+
+   # in my case (username is lwander), the above command looks like:
+   # git clone git@github.com:lwander/halyard.git
+
+cd halyard
+
+git remote add-url upstream git@github.com:spinnaker/halyard.git
+
+  # the above command allows you to push patches to the upstream repo for 
+  # approval
+```
+
+## Configuring IntelliJ
```

Comment:
This is for your benefit mostly so you can do as you please.

FWIW github seems to support my claim in their "standard" recommendation
https://help.github.com/articles/setting-guidelines-for-repository-contributors/